functions/src/create_friend_invitation.ts

```diff
@@ -106,7 +110,8 @@ export const createFriendInvitation = functions.https.onRequest(
           .where("sender", "==", senderId)
           .where("friend", "==", friendId)
           .where("term", "==", term)
-          .where("version", "==", version)
+          .where("versions", "==", versions)
```

Comment:
is firebase able to compare arrays like this ?

Reply:
yes, I tested and if I put in same versions it does delete the old record